docs/user/README.md

```diff
@@ -29,23 +34,22 @@ Supported integration scenarios are neutral to the vendor of the target system.
 
 Currently, logs are based on the Fluent Bit protocol. If you're curious about the progress to switch to the vendor-neutral OTLP protocol, follow this [epic](https://github.com/kyma-project/kyma/issues/16307).
 
-
 ## Components
 
-![Components](./assets/general-components.drawio.svg)
+![Components](./assets/telemetry-components.drawio.svg)
```

Comment:
The reference key should be placed below the diagram.
Also, it may be useful to explain the purpose of the dashed connectors/lines/boundaries in the reference key. The explanation in the guidelines is just an example.

Reply:
Thank you! I got rid of some of the dashed lines and explained the others. 
Reference key is below the diagram now.